src/dotnet/APIView/APIView/Model/V2/ReviewToken.cs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace APIView.Model.V2
+{
+    /// <summary>
+    /// Token corresponds to each component within a code line. A separate token is required for keyword, punctuation, type name, text etc.
+    /// </summary>
+    public class ReviewToken
+    {
+        public ReviewToken() { }
+        public ReviewToken(string value, TokenKind kind)
+        {
+            Value = value;
+            Kind = kind;
+        }
+        public TokenKind Kind { get; set; }
+        public string Value { get; set; }
+        /// <summary>
```

Comment:
nit: spacing above comments

```suggestion
        public string Value { get; set; }

        /// <summary>
```